File: face_recognition_api/app/model/user_model.py
from sqlalchemy import  Column, Integer, String
from sqlalchemy.orm import relationship

from database import base

class User(base):
  __tablename__ = "user"

  user_id = Column(Integer, primary_key=True, index=True)
  user_name = Column(String(255),nullable=True, unique=True)
  email = Column(String(255),nullable=False, unique=True)
  nim = Column(Integer, nullable=False, unique=True)
  password = Column(String(255), nullable=False)
  face_image = Column(String(50), nullable=True)

  attendance = relationship("Attendance", back_populates="user")

File: face_recognition_application/lib/api/model/user_model.dart
import 'dart:io';

class UserModel {
  int? userId;
  String? userName;
  String? email;
  int? nim;
  File? faceIamge;
  String? token;
  String message;

  UserModel(
      {this.userId,
      this.userName,
      this.email,
      this.nim,
      this.faceIamge,
      this.token,
      required this.message});
}
